.github/workflows/pr-title-linear.yml

```
@@ -0,0 +1,65 @@
+                `**Expected format:** \`10N-123: Your PR title\`\n` +
+                `**Current title:** \`${prTitle}\`\n\n` +
+                `Please update your PR title to include the Linear issue ID at the beginning.`;
+              
+              // Post comment if title is invalid
+              await github.rest.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                body: errorMessage
+              });
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)